core/common/src/main/java/alluxio/LeaderInquireClient.java

```diff
@@ -106,7 +106,8 @@ public synchronized String getMasterAddress() {
         CommonUtils.sleepMs(LOG, Constants.SECOND_MS);
       }
     } catch (Exception e) {
-      LOG.error("Error with zookeeper: {} message: {}", mZookeeperAddress, e.getMessage());
+      LOG.error("Error getting the master address from zookeeper. Zookeeper address: {}",
+          mZookeeperAddress, e.getMessage());
```

Comment:
When the last argument to `LOG.error` is a Throwable, the logger will print its stack trace, including the error message. It will look like
```
Error getting the master address from zookeeper. Zookeeper address: alluxiomaster
{fully qualified class name of exception}: {output of e.getMessage()}
{exception stack trace}
```